peachjam/templates/peachjam/_help_popover.html

```diff
@@ -0,0 +1,7 @@
+<div class="mb-3">
+  <a href="#{{ link }}"
```

Comment:
@longhotsummer here is the `#` referenced in the above comment

Reply:
The page that embeds this template can provide the path to the page eg `/judgments/courts` and then we can hardcode the domain.